### ISSUE TITLE
Exported db data of JavaScript milliseconds is not represented correctly

### ISSUE THREAD

**Issue by michelpetrovic** [jepiqueau/capacitor-sqlite#24]
I store JavaScript milliseconds (e.g. `Date.now()`) in a field defined as:
`starttime INTEGER NOT NULL`

Before the export the value is stored correctly:
`1590952516107` == `Sun May 31 2020 21:15:16 GMT+0200 (Central European Summer Time)`

When exported, the value is incorrectly represented as:
`1814616587` == `Thu Jan 22 1970 01:03:36 GMT+0100 (Central European Standard Time)`

Looking at the `capacitor-sqlite` source code, it seems that Sqlite `INTEGER` is stored in a Java `Integer`, which results in an overflow.
The following test confirms this:
```
int i = (int) 1590952516107L;
System.out.println("i = " + i); // => i = 1814616587
```

Given that [Sqlite's `INTEGER` can store up to 8 bytes](https://www.sqlite.org/datatype3.html#storage_classes_and_datatypes), the Java data type should be `Long`, not `Integer`.

**Comment by jepiqueau** [comment]
@michelpetrovic Thanks for using the plugin, this will be fixed in the next release 2.1.0-7 as soon as i can publish it successfully on npm.

**Comment by michelpetrovic** [comment]
Thank you for the quick fix, everything works correctly now.

And thank you for sharing the plugin!
